include/alpaka/extent/Traits.hpp

```diff
@@ -22,121 +23,183 @@ namespace alpaka
         //!
         //! If not specialized explicitly it returns 1.
         template<typename TIdxIntegralConst, typename TExtent, typename TSfinae = void>
-        struct GetExtent
-        {
-            ALPAKA_NO_HOST_ACC_WARNING
-            ALPAKA_FN_HOST_ACC static auto getExtent(TExtent const&) -> Idx<TExtent>
-            {
-                return static_cast<Idx<TExtent>>(1);
-            }
-        };
-
-        //! The extent set trait.
-        template<typename TIdxIntegralConst, typename TExtent, typename TExtentVal, typename TSfinae = void>
-        struct SetExtent;
+        struct [[deprecated("Specialize GetExtents instead")]] GetExtent{
+            ALPAKA_NO_HOST_ACC_WARNING ALPAKA_FN_HOST_ACC static auto getExtent(TExtent const&)
+                ->Idx<TExtent>{return static_cast<Idx<TExtent>>(1);
     } // namespace trait
+}; // namespace alpaka
```

Comment:
This comment and the comment above is wrong!

Please remove the comments.

Reply:
Maybe squashing this segment to `namespace alpaka::trait` improves things a bit...